_docs/customer-stories/epaairnow.md

```diff
@@ -0,0 +1,55 @@
+---
+parent: customer-stories
+layout: docs
+sidenav: true
+title: Environmental Protection Agency AirNow
+kicker: Case study
+redirect_from:
+  - /overview/customer-stories/environmental-protection-agency-airnow/
+  - /case-studies/epaairnow/
```

Comment:
```suggestion
  - /case-studies/epa-airnow/
```

It's best to separate words by either a hypen or a slash to improve search engine optimization and searching by distinct terms like "epa" and "airnow" to find this post